### PR TITLE
change tag pdf

### PR DESCRIPTION
**Pull request by mdurrani808**

### CODE REVIEW

**Comment by mdurrani808** [comment]
Wait to merge until someone takes one for the team and prints one of these out to make sure you dont get 15 blurry messes.

File: source/docs/getting-started/april-tags.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ AprilTags are a type of visual fiducial marker that is commonly used within robo
 
 A more technical explanation can be found in the `WPILib documentation <https://www.google.com>`_.
 
-.. note:: You can get a PDF of properly sized 16h5 targets :download:`here<assets/alltags.pdf>`.
+.. note:: You can get a PDF of properly sized 16h5 targets `here <https://github.com/Tigerbotics7125/AprilTag16h5>`_.
```
Comment (shueja-personal) [inline source/docs/getting-started/april-tags.rst:17]: Why does this still link to Tigerbotics instead of assets?

Comment (mdurrani808) [reply source/docs/getting-started/april-tags.rst:17]: Gives teams a bit more flexibility, has PNGs of the tags and printing instructions as well.

**Review by shueja-personal — APPROVED** [review]

**Review by gerth2 — APPROVED** [review]
Jeremiah said good so I say good

**Pull request merged**